sdk/tests/utilities/test_api_configuration_loader.py

```diff
@@ -178,11 +178,8 @@ def test_load_from_environment_vars_only(self):
 
     def test_missing_secrets_file_logs_message_at_debug(self):
 
-        with self.assertLogs() as captured:
-            import logging
-            logger = logging.getLogger()
-            logger.setLevel(logging.DEBUG)
-
+        with self.assertLogs(level="DEBUG") as captured:
```

Comment:
this was failing locally for me, not sure why, but this change is also more "correct"

Reply:
that's much better.  I tried removing some of those logging things and everything broke.  Asserts on the logging output. :(